### PR TITLE
Move examples to subdir

### PR DESCRIPTION
**Pull request by eochoa260**
Moved each example into its own file and updated readme.

### CODE REVIEW

File: examples/client.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/*
+Wrapping the client can be useful when you want to override default behavior, such as always setting
```
Comment (eochoa260) [inline examples/client.go:2]: I wasn't sure if we wanted to retain this comment in the root readme, create a new readme in the examples dir?

File: examples/mutation.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
+var mutation struct {
```
Comment (eochoa260) [inline examples/mutation.go:1]: Should I be importing packages in these files?  I just copied what was in the readme for now.

Comment (rocktavious) [reply examples/mutation.go:1]: I think this would be helpful context to have in the example - i would add it

File: examples/client.go (hunk 1 above)
Comment (rocktavious) [reply examples/client.go:2]: lets go with this for now - if someone is digging through example code having info inline like this is probably what they'd expect

**Review by rocktavious — COMMENTED** [review]
Lets make the imports and package name changes and call it good.

**Review by rocktavious — APPROVED** [review]
LGTM

**Pull request merged**